.eslintrc.cjs

```diff
@@ -12,7 +12,7 @@ module.exports = {
     tsconfigRootDir: __dirname,
   },
   root: true,
-  "ignorePatterns": [".eslintrc.cjs"],
+  ignorePatterns: ['.eslintrc.cjs'],
```

Comment:
I did not expect this formatting.

Reply:
I think my eslint just auto-formatted this, but the format looks in tandem with others so maybe that's ok?

Reply:
sigh...

Our settings are not the same for some reason.